api/db/knex.js

```diff
@@ -1,6 +1,13 @@
 const knex = require('knex');
 const config = require('../knexfile');
 
-const db = process.env.NODE_ENV ? knex(config[process.env.NODE_ENV]) : () => {};
+const { NODE_ENV } = process.env;
+if (!NODE_ENV) {
+  let msg = "❌ NODE_ENV is not set, unable to determine knex configuration\n";
+  msg += "Please set NODE_ENV to 'development', 'test', or 'production'\n";
+  msg += "Terminating...";
+  console.error(msg);  /* eslint-disable-line no-console */
+  process.exit(1);
+}
```

Comment:
Added this error condition around `NODE_ENV`, so that we don't continue executing if we don't know what knex configuration to use. This should help when troubleshooting.